src/org/thoughtcrime/securesms/profiles/AvatarHelper.java

```diff
@@ -61,35 +61,29 @@ public static void setGroupAvatar(Context context, int chatId, Bitmap bitmap) {
         }
     }
 
-    public static InputStream getInputStreamFor(@NonNull Context context, @NonNull String address)
+    public static InputStream getInputStreamFor(@NonNull Context context)
```

Comment:
What cought my eye was this method here. It is called `getInputStreamFor`, which sounds as if it could also be used to get a "foreign" avatar, like `getInputStreamFor(context, "bob@testrun.org").` (if `self` is Alice). Of course, before this PR, this worked just fine because `getSelfAvatarFile(context, "bob@testrun.org") returned the avatar of Bob, not the self-avatar. 

I looked around both usages actually meant to get the self-avatar, so no risk here. I think that it should be inlined, however, because this method does not do anything actually useful and the name is very confusing.